integrations/terraform/examples/resources/teleport_integration/resource.tf

```
@@ -0,0 +1,69 @@
+resource "teleport_integration" "aws_oidc" {
+  version  = "v1"
+  sub_kind = "aws-oidc"
+  metadata = {
+    name        = "example"
+    description = "AWS OIDC integration"
+    labels = {
+      env = "dev"
+    }
+  }
+
+  spec = {
+    aws_oidc = {
+      issuer_s3_uri = "s3://example-bucket/some-prefix"
+      role_arn      = "arn:aws:iam::123456789012:role/example-role-name"
+    }
+  }
+}
+
+resource "teleport_integration" "azure_oidc" {
+  version  = "v1"
+  sub_kind = "azure-oidc"
+  metadata = {
+    name        = "azure-oidc"
+    description = "Example Azure OIDC integration"
+    labels = {
+      env = "dev"
+    }
+  }
+
+  spec = {
+    azure_oidc = {
+      // Azure Entra ID tenant ID
+      tenant_id = "a1b2c3d4-f2e4-97a8-9abc-1234567890ab"
+      // Azure enterprise application client ID
+      client_id = "7f12e3b5-6789-4abc-def0-112233445566"
+    }
+  }
+}
+
+resource "teleport_integration" "aws_roles_anywhere" {
+  version  = "v1"
+  sub_kind = "aws-ra"
+  metadata = {
+    name        = "aws-ra"
+    description = "Example AWS Roles Anywhere integration"
+    labels = {
+      env = "dev"
+    }
+  }
+
+  spec = {
+    aws_ra = {
+      profile_sync_config = {
+        // sync AWS profiles as Teleport applications
+        enabled                           = true
+        profile_accepts_role_session_name = false
+        profile_arn                       = "arn:aws:rolesanywhere:us-east-1:123456789012:profile/example-profile-name"
```

Profile ARN ends with the uuid for the profile, and not its name.
Mostly a nit, but keeping example data as close to real data as possible should help on comprehension.


```suggestion
        profile_arn                       = "arn:aws:rolesanywhere:us-east-1:123456789012:profile/<random-uuid>"
```